src/schema/fair.ts

```diff
@@ -26,6 +28,18 @@ import ShowSort from "./sorts/show_sort"
 import { allViaLoader } from "lib/all"
 import { FairArtistSortsType } from "./sorts/fairArtistSorts"
 
+const FollowedContentType = new GraphQLObjectType({
+  name: "followed_content",
```

Comment:
Minor: The type name should be `FollowedContent` to be consistent with everything else (you can see in the SDL how it looks a bit weird `followed_content: followed_content`)